eclipsePlugin/build.gradle

```diff
@@ -18,28 +18,36 @@ sourceSets {
 
 configurations {
   embeddedLibs
+  includedLibs
   compile.extendsFrom embeddedLibs
+  compile.extendsFrom includedLibs
 }
 
 def localProps = new Properties()
 localProps.load(new FileInputStream("$projectDir/local.properties"))
 
-def requiredLibs = fileTree(dir:'lib', include:['jsr305.jar',
+// TODO : convert these to external dependencies
+def requiredLibs = fileTree(dir:'lib', include:[
```

Comment:
I probably could if I changed the eclipse dependencies from `compile` to `compileOnly`, although I would have to make sure to filter dependencies from extended configurations (`compileOnly`, `embeddedLibs`) when creating the distribution jars / zips...

Probably better to address that as we change those dependencies to actual maven dependencies instead of local libs...